physical/mysql.go

```diff
@@ -0,0 +1,190 @@
+package physical
+
+import (
+	"database/sql"
+	"errors"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/armon/go-metrics"
+	_ "github.com/go-sql-driver/mysql"
+)
+
+var (
+	MySQLDBNameMissing          = errors.New("database name is missing in the configuration")
```

Comment:
You probably only need to put the errors that are used more than once at this level. The single use errors can just be defined at the call site.